### PR TITLE
Migrate to bot account

### LINKED ISSUE #63: Migrate Jitsi plugin to use bot accounts
If you're interested please comment here and come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Plugin: Jitsi" community channel](https://community-release.mattermost.com/core/channels/plugin-jitsi).

New contributors please see our [Developer's Guide](https://developers.mattermost.com/contribute/getting-started/) and our [Plugins Guide](https://developers.mattermost.com/extend/plugins/).

----

Migrate Jitsi plugin to use bot accounts.

For an example, here is a PR that migrated the Demo Plugin to use bot accounts with a helper function https://github.com/mattermost/mattermost-plugin-demo/pull/29

### PR DESCRIPTION
**Pull request by nathanaelhoun**
#### Summary

- [x] Use a bot account to send ephemeral messages

#### Ticket Link

Closes #63.

### CODE REVIEW

File: server/plugin.go [hunk 1]
```diff
@@ -57,6 +59,22 @@ func (p *Plugin) OnActivate() error {
 	}
 	p.i18nBundle = i18nBundle
 
+	jitsiBot := &model.Bot{
+		Username:    "jitsi",
+		DisplayName: "Jitsi",
+		Description: "A bot account created by the jitsi plugin",
+	}
+	options := []plugin.EnsureBotOption{
+		plugin.ProfileImagePath("assets/icon_square.png"),
```
Comment (hanzei) [inline server/plugin.go:68]: Maybe use `icon.png` as filename to indicate that it's the same as https://github.com/mattermost/mattermost-plugin-jitsi/blob/master/assets/icon.svg?

Comment (nathanaelhoun) [reply server/plugin.go]: Done :white_check_mark:

File: webapp/src/manifest.ts [hunk 2]
```diff
@@ -107,6 +99,14 @@ const manifest = JSON.parse(`
                 "help_text": "(Optional) The number of minutes from when the meeting link is created to when it becomes invalid. Minimum is 1 minute. Only applies if using JWT authentication for your Jitsi server.",
                 "placeholder": "",
                 "default": 30
+            },
```
Comment (jespino) [inline webapp/src/manifest.ts:102]: This comes from other PR, but it is ok to go here.

File: server/plugin.go [hunk 3]
```diff
@@ -57,6 +59,22 @@ func (p *Plugin) OnActivate() error {
 	}
 	p.i18nBundle = i18nBundle
 
+	jitsiBot := &model.Bot{
+		Username:    "jitsi",
+		DisplayName: "Jitsi",
+		Description: "A bot account created by the jitsi plugin",
+	}
+	options := []plugin.EnsureBotOption{
+		plugin.ProfileImagePath("assets/icon.png"),
```
Comment (jespino) [inline server/plugin.go:68]: The icon that we have is an `svg`

Comment (jespino) [reply server/plugin.go:68]: I think is ok to have a generate png version, but you need to add that to the commit

Comment (hanzei) [reply server/plugin.go:68]: As far as I know a profile picture has to be an `png`, not `svg`.

Comment (nathanaelhoun) [reply server/plugin.go:68]: Yes, `svg` profile picture was not accepted. But I forgot to add it to the commit :facepalm: This is fixed!

**Comment by nathanaelhoun** [comment]
Thanks for having fixed the tests @jespino! I was trying to understand the origin of this, as I am learning to do tests, and I was blocked

File: webapp/src/manifest.ts (hunk 2 above)
Comment (nathanaelhoun) [reply webapp/src/manifest.ts:102]: Thanks :+1:

**Comment by hanzei** [comment]
@DHaussermann Can we test this in release testing?

**Pull request merged**